.github/workflows/linter client.yml

```diff
@@ -0,0 +1,39 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Client Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [master]
+    # Remove the line above to run when pushing to master
+  pull_request:
```

Comment:
Just for my clarification, because I am not sure how it handles it. When I push something and create a PR, the linter is run twice right? And where do I see the results if it runs on a push? 
Wouldn't it be easier to just check on PRs to the DEV and have that as a definitive requirement to pass, then each developer can run it locally and take care of it going through with the command you created?

Reply:
> When I push something and create a PR, the linter is run twice right?
So basically when you do a PR, the linter will run twice. Thats true. But after that, whenever the code in the PR is changed, it will run again. Therefore a on:pull_request would not be enough. I think the double linting, when creating a pr is okay. The annotations do not double.

Writing the events with : at the end, is a long version of an array like on:[push, pull_request]. See the docs [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events), if you want to read more into it. 

>  And where do I see the results if it runs on a push?
The annotations in the PR Diffs will be changed

> Wouldn't it be easier to just check on PRs to the DEV
Like I've written, I think the check when new changes ar pushed are good aswell. I would not trust each dev to do it on their own machine, hence that's why we have the bot.